.github/workflows/eden-trusted.yml

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) 2025, Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+---
+name: Run Eden
+
+on:  # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows: ["PR Gate"]
+    types: [completed]
```

Comment:
why not trigger the workflow from the gate and pass the parameters to avoid parsing JSON files?